### PR TITLE
Feature/email secretary after decisions

### PR DESCRIPTION
**Pull request by EdoStorm96**
Some fixes for a two small issues. 

the fix for #581 is pretty self explanatory.

For the issue #563, I updated the `update_go()` of `Review`, as Ty suggested. Please check if this is in the correct spot ... In my testing this works as expected.

Furthermore, I'd appreciate feedback on the text of the e-mail. It's pretty bare bones, but gets the job done, with an overview of all decisions.

PS. I accidentally blacked review_utils.py, but undid it, because it made the diff unreadable.

### CODE REVIEW

File: reviews/utils/review_utils.py [hunk 1]
```diff
@@ -304,6 +304,21 @@ def notify_secretary(decision):
...
+    subject = _("FETC-GW {}: alle beoordelingen toegevoegd").format(
+        review.proposal.committee_prefixed_refnum(),
+    )
+    params = {
+        "secretary": secretary.get_full_name(),
+        "proposal": review.proposal,
+        "decisions": review.decision_set.all()
+    }
+    msg_plain = render_to_string("mail/all_decisions_notify.txt", params)
+    send_mail(subject, msg_plain, settings.EMAIL_FROM, [secretary.email])
```
Comment (tymees) [inline reviews/utils/review_utils.py:321]: As per #563:
> [..] FETC npm (not users!) [..]

Thus, the recipient should be the general fetc email ;)

File: reviews/utils/review_utils.py [hunk 2]
```diff
@@ -304,6 +304,21 @@ def notify_secretary(decision):
...
     send_mail(subject, msg_plain, settings.EMAIL_FROM, [secretary.email])
 
+def notify_secretary_all_decisions(review):
+    """
+    Notifies a secretary all Decisions have been made for a certain review
+    """
+    secretary = get_secretary()
+    subject = _("FETC-GW {}: alle beoordelingen toegevoegd").format(
+        review.proposal.committee_prefixed_refnum(),
+    )
```
Comment (tymees) [inline reviews/utils/review_utils.py:314]: You're translating the string, but not adding any translations...

Honestly, as the email is not translated, I'd just drop the `_([..])` tag altogether

**Review by tymees — CHANGES_REQUESTED** [review]
Nice quick turnaround lol :)

Two small errors, otherwise: looks great!

About the email: looks fine; during the meeting we all agreed the contents should not matter that much, the subject is more important.

File: reviews/utils/review_utils.py [hunk 3]
```diff
@@ -304,6 +304,21 @@ def notify_secretary(decision):
...
+    secretary = get_secretary()
+    subject = "FETC-GW {}: alle beoordelingen toegevoegd".format(
+        review.proposal.committee_prefixed_refnum(),
+    )
+    params = {
+        "secretary": secretary.get_full_name(),
+        "proposal": review.proposal,
+        "decisions": review.decision_set.all()
+    }
+    msg_plain = render_to_string("mail/all_decisions_notify.txt", params)
```
Comment (miggol) [inline reviews/utils/review_utils.py:320]: Any translations existing in this input, e.g. `decision.get_go_display`, will render to the currently selected language.

If the user who triggered the `update_go()` happened to be set to English, all translations in here will also turn into English, which is a bit weird. 

See `proposal_utils.notify_local_staff()` for an example of how we manually activate a language for email sending, and then reset it afterwards.

**Review by miggol — CHANGES_REQUESTED** [review]

File: reviews/utils/review_utils.py [hunk 4]
```diff
@@ -304,6 +304,33 @@ def notify_secretary(decision):
...
+        "review": review,
+        "decisions": review.decision_set.all(),
+        "review_detail_page": settings.BASE_URL
+        + (reverse("reviews:detail", args=[review.pk])),
+        "close_review_page": settings.BASE_URL
+        + (reverse("reviews:close", args=[review.pk])),
+    }
+    msg_plain = render_to_string("mail/all_decisions_notify.txt", params)
+    send_mail(subject, msg_plain, settings.EMAIL_FROM, ["fetc-gw@uu.nl"])
+
```
Comment (tymees) [inline reviews/utils/review_utils.py:330]: I should've clarified; you should use `settings.EMAIL_FROM` as the recipient.

It's kinda weird, I know, but `EMAIL_FROM` is the NPM email. Hardcoding this is a bad idea, as names (and thus, emails) in the UU tend to change. It's better if we only have to change the one value when that happens

**Review by tymees — CHANGES_REQUESTED** [review]
One tiny tweak left, and then it's GTG

You don't have to re-request review from me if you fix that tiny detail

**Pull request merged**